docs/docusaurus/i18n/en.json

```diff
@@ -936,16 +951,24 @@
       "Community": "Community"
     },
     "categories": {
+      "Getting Started": "Getting Started",
+      "Tutorials": "Tutorials",
+      "Usage": "Usage",
+      "Architecture": "Architecture",
+      "Guide: Deploy": "Guide: Deploy",
```

Comment:
I dislike seeing "Guide:" on the sidebars, It might be better for us to either call this 'Deploy', 'Architecture', 'Usage' itself or have a subsection under guide. Since we may have more guides in the future, I would probably go with latter.

Reply:
The latter was the original intent. Unfortunately Docusaurus doesn't support two-tiered subcategorization, so we can't do e.g. Guides > Deploy > Orchestrator. I think for now we can just remove the `Guide: ` and, if the number of guides gets too large, we can figure out a solution then